libs/patchwheel/patch.go

```diff
@@ -0,0 +1,280 @@
+package patchwheel
+
+import (
+	"archive/zip"
+	"bufio"
+	"bytes"
+	"context"
+	"crypto/sha256"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+const (
+	versionKey = "Version:"
+	nameKey    = "Name:"
+)
+
+// readMetadataAndRecord scans the zip file for files matching the patterns
+// "*.dist-info/METADATA" and "*.dist-info/RECORD". If multiple are found, it picks
```

Comment:
Why would there be multiple files like this?

Reply:
And if there are multiple ones, what if METADATA one is for one version and RECORD is for different one?

Reply:
Good questions. I simplified this and removed glob matching.